packages/api/src/controllers/locationController.js

```diff
@@ -66,7 +66,7 @@ const LocationController = {
         const result = await LocationModel.transaction(async trx => {
           return await LocationModel.query(trx).context({ user_id: req.user.user_id }).upsertGraph(
             { ...req.body, location_id: req.params.location_id },
-            { noInsert: true, noDelete: true, noUpdate: nonModifiable });
+            { noInsert: [...figures, ...assets, 'figure'], noDelete: true, noUpdate: nonModifiable });
```

Comment:
a new organic_history entry will be created when req.body include organic_history
req.body: {...location, field: {...field, organic_history: {organic_status, effective_date}}}
We can also remove organic_history from req.body and add it in post/put endpoints, but effective date might be off by one day because of timezone difference